### ISSUE TITLE
InvalidPathException on "colon" character in a windows path

### ISSUE THREAD

**Issue by efrancis59** [awslabs/djl#526]
## Description
I'm trying to launch the **SentimentAnalysis** example. But I get an exception generated by SimpleVocabulary.Builder that tries to use the vocabulary path containing a "C:" prefix.

### Expected Behavior
This class is expected to work under windows also.

### Error Message
`
Exception in thread "main" java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/xyz/.djl.ai/cache/repo/model/nlp/sentiment_analysis/ai/djl/pytorch/distilbert/distilbert/sst/0.0.1/distilbert-base-uncased-finetuned-sst-2-english-vocab.txt
        at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
        at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
        at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:229)
        at java.base/java.nio.file.Path.of(Path.java:147)
        at java.base/java.nio.file.Paths.get(Paths.java:69)
        at ai.djl.modality.nlp.SimpleVocabulary$Builder.addFromTextFile(SimpleVocabulary.java:219)
        at ai.djl.pytorch.zoo.nlp.sentimentanalysis.PtDistilBertTranslator.prepare(PtDistilBertTranslator.java:55)
        at ai.djl.inference.Predictor.batchPredict(Predictor.java:148)
        at ai.djl.inference.Predictor.predict(Predictor.java:128)
        at com.et.ai.SentimentAnalysis.predict(SentimentAnalysis.java:64)
        at com.et.ai.SentimentAnalysis.main(SentimentAnalysis.java:45)
`

## How to Reproduce?
Just try to launch a copy of the **SentimentAnalysis** class under windows. The used **djl** version is 0.9.

**Comment by frankfliu** [comment]
@efrancis59 Woud you please provide your code?

This issue should have been fixed in 0.9.0 release. I tried the following on Windows and it's working fine:
```
Criteria<String, Classifications> criteria = Criteria.builder()
    .optApplication(Application.NLP.SENTIMENT_ANALYSIS)
    .setTypes(String.class, Classifications.class)
    .optProgress(new ProgressBar())
    //.optModelPath(Paths.get("c:/Users/Administrator/.djl.ai/cache/repo/model/nlp/sentiment_analysis/ai/djl/pytorch/distilbert/distilbert/sst/0.0.1"))
    .optModelUrls("file:///c:/Users/Administrator/.djl.ai/cache/repo/model/nlp/sentiment_analysis/ai/djl/pytorch/distilbert/distilbert/sst/0.0.1")
    .optModelName("traced_distilbert_sst_english")
    .optTranslator(new PtDistilBertTranslator())
    .optDevice(Device.cpu())
    .build();
```

I tried `.optModelPath()` and `.optModelUrls()`,  both working fine for me.

**Comment by frankfliu** [comment]
@efrancis59 Sorry, my bad, the fix is after 0.9.0
You can try 0.10.0-SNAPSHOT version: https://github.com/awslabs/djl/pull/467

**Comment by frankfliu** [comment]
Feel free to re-open this issue if you still have problem with 0.10.0-SNAPSHOT version.
